static/js/controllers/contacts-report.js

```diff
@@ -62,7 +70,7 @@ angular.module('inboxControllers').controller('ContactsReportCtrl',
 
       $scope.enketoStatus.saving = true;
       $scope.enketoStatus.error = null;
-      Enketo.save($state.params.formId, $scope.form)
+      Enketo.save($state.params.formId, $scope.form, null, geolocation)
```

Comment:
Make `geolocation` the 3rd parameter and reportId the 4th so you don't have to pass null.